resources/geocoding/es/54.txt

```diff
@@ -14,225 +14,311 @@
 
 # Generated from:
 # http://www.cnc.gov.ar/infotecnica/numeracion/indicativosinter.asp (2011-07-11)
-# http://en.wikipedia.org/wiki/+54 [431727225]
+# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina
 # Some manual edits were made after the file was generated, according to the
 # ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
```

Comment:
Remove these 4 lines and please replace it with below lines as below,

\# Historically generated from:
\# http://www.cnc.gov.ar/infotecnica/numeracion/indicativosinter.asp (2011-07-11)
\# http://en.wikipedia.org/wiki/+54 [431727225]
\# Some manual edits were made after the file was generated, according to the
\# ITU file published 2011-12-16 here: http://www.itu.int/oth/T0202000009/en
\# Recently updated by consulting 
\# https://es.wikipedia.org/wiki/N%C3%BAmeros_telef%C3%B3nicos_en_Argentina